common/src/main/java/com/wynntils/functions/LootrunFunctions.java

```diff
@@ -121,6 +122,24 @@ public FunctionArguments.Builder getArgumentsBuilder() {
         }
     }
 
+    public static class LootrunMissionFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            int missionIndex = arguments.getArgument("index").getIntegerValue();
+
+            MissionType mission = Models.Lootrun.getMission(missionIndex);
+            if (mission == null) return "§cNone";
+
+            return mission.getColoredName();
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.RequiredArgumentBuilder(
+                    List.of(new FunctionArguments.Argument<>("index", Integer.class, -1)));
```

Comment:
```suggestion
                    List.of(new FunctionArguments.Argument<>("index", Integer.class, 
null)));
```

Required arguments have null default values.